src/nbl/video/COpenGLFeatureMap.h

```diff
@@ -518,6 +520,7 @@ class COpenGLFeatureMap
 	"GL_KHR_texture_compression_astc_ldr",
 	"GL_KHR_blend_equation_advanced",
 	"GL_KHR_blend_equation_advanced_coherent",
+	"GL_KHR_shader_subgroup",
```

Comment:
actually that extensions adds quite a bit of OpenGL queries, I just noticed while trying to test example 48 with native funcs